configuration/etl/etl_action_defs.d/hardware/hardware_types.json

```diff
@@ -8,7 +8,7 @@
             "system_id": "st.id",
             "cpu_id": "ct.id",
             "core_count": "s.core_count",
-            "memory_gb": "CEILING(s.physmem / 1024.0)",
+            "memory_gb": "s.physmem",
```

Comment:
Please can you confirm that this change is intentional. Where does the memory value normalization occur now?

Reply:
Yes, the memory gets converted in the 'transform' step of the SUPReMM archive reader (`get_hardware_info.py`), although it (temporarily) is also taken care of by the SUPReMM patching script, so that data extracted before this change also gets converted.